docs/UserGuide.md

```diff
@@ -1,134 +1,162 @@
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
-
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+#### Adding a task: `add`
+To add a new task to Taskell, use the add command.<br>
+Formats: 
+- `add TASK ITEM` <br>
+This format can be used to add floating tasks without any deadlines.<br>
+> Example: `add read Harry Potter Book` <br>
+- `add TASK ITEM by [DATE]` <br>
+This format can be used to add events held on a certain day.<br>
+> Example: `add buy MA1101R textbook by today` <br>
+> Example: `add do CS2100 assignment by 10th August`<br>
+> Example: `add go shopping by tomorrow`<br>
+- `add TASK ITEM by [DATE][TIME]` <br>
+This format can be used to add tasks with a stipulated deadline.<br>
+> Example: `add do lab homework on Friday 7pm` <br>
+-  `add TASK ITEM on [DATE]` <br>
+The use of word �on� gives more flexibility for you.<br>
+> Example: `add schedule meeting on Thursday` <br>
+- `add TASK ITEM on [DATE][TIME]` <br>
+> Example: `add meet teacher on Friday 7pm` <br>
+
+> Take Note! Dates need to refer to the current week dates the task was keyed in. Eg. 
+> "tomorrow" and "thursday" is valid but "next Saturday" is invalid. Only the words "on"
+> or "by" has to be used to distinguish between dateline and task description.<br>
+
+#### Listing all tasks : `list`
+To view a list of all the tasks, <kbd>Enter</kbd> list.<br>
+<br>
+Formats: 
+- `list` <br>
+Prints a list of all the uncompleted tasks.<br>
+- `list DATE` <br> 
+Prints a list of all the completed tasks.<br>
+- `list DONE` <br>
+Prints a list of all the tasks due on the specific date.<br>
+<br><img src="images/ListToday.png" width="600"></br>
+Picture 2: List of tasks due today printed when �List today� is keyed in.
+   
+#### Finding tasks: `find`
+To view tasks with similar keyword, use the find command.<br>
+Formats: <br>
+-`find KEYWORD [MORE_KEYWORDS]`<br>
+Prints a list of activities that match the keywords.<br>
+Example: `find banana milk essay:`<br>
+This returns any task description having keywords banana, milk, essay. <br>
```

Comment:
try to use a different font/command formatting of markdown etc., to indicate words related to / used in the commands for easy distinguishing :-)